src/components/TokenEnterAmount.tsx

```diff
@@ -51,21 +51,14 @@ export function formatNumber(value: string) {
     .replaceAll('_', groupingSeparator)
 }
 
+/**
+ * Unformats value from the phone's selected number format to a standard "1234.5678" js number.
+ * This function IS NOT indended for:
+ *   - for numbers that don't follow the format from phone's settings
+ *   - external numbers that are already in JS format (e.g from API responses)
```

Comment:
is this comment true? if i understand correctly, the problem isn't about external numbers or the JS format, but rather that external numbers _always_ come in JS format (for now) and that is only a problem if the device is _not_ using the JS format for the number settings? perhaps it'd be more clear if we didn't identify this as a separate issue 
```suggestion
 *   Do not use this function with values that are not formatted according to the device number format settings, e.g. values that are returned by API's in the standard js format.
```